contessa/migration.py

```diff
@@ -0,0 +1,120 @@
+# Python script that will apply the migrations up to head
+from contessa.db import Connector
+from packaging.version import parse as pv
+
+ALEMBIC_TABLE = "alembic_version"
```

Comment:
Please make sure that this works also in the case when the contessa is only dependency of the main project, which also uses Alembic, and stores everything in one database (e.g. Airflow running locally). 
I would use our unique name, e.g. `"contessa_version"`. 
This also applies to the config file and paths to scripts and migration files.

Reply:
very good point, I changed all contessa migrations tables to name 'contessa_alembic_version'